rails_multitenant.gemspec

```
@@ -26,8 +26,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 3.1'
 
-  spec.add_dependency 'activerecord', '>= 7.0', '< 7.3'
-  spec.add_dependency 'activesupport', '>= 7.0', '< 7.3'
+  spec.add_dependency 'activerecord', '>= 7.0'
+  spec.add_dependency 'activesupport', '>= 7.0'
```

As per our new guideline, we should have an upper bound.

```suggestion
  spec.add_dependency 'activerecord', '>= 7.0', '< 8.1'
  spec.add_dependency 'activesupport', '>= 7.0', '< 8.1'
```